(ForumsTeam) Spelling mistake 'ah' need to modify 'to an'

AjayKumar-N opened the pull request:
The referenced doc: https://docs.microsoft.com/en-us/azure/app-service/app-service-plan-manage#scale-an-app-service-plan
Under the section "Scale an App Service plan" > there is a spelling mistake 'ah', need to modify to 'an'

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

AjayKumar-N committed:
(ForumsTeam) Spelling mistake 'ah' need to modify 'to an'

The referenced doc: https://docs.microsoft.com/en-us/azure/app-service/app-service-plan-manage#scale-an-app-service-plan
Under the section "Scale an App Service plan" > there is a spelling mistake 'ah', need to modify to 'an'

GitHubber17 committed:
Update app-service-plan-manage.md
